debian/control

```diff
@@ -0,0 +1,11 @@
+Source: ocf-utils
+Section: python
+Priority: extra
+Maintainer: Open Computing Facility <help@ocf.berkeley.edu>
+Build-Depends: debhelper (>= 9), python, dh-virtualenv (>= 0.8)
+Standards-Version: 3.9.6
+
+Package: ocf-utils
+Architecture: any
+Depends: ${shlibs:Depends}, virtualenv | python-virtualenv
```

Comment:
Do we need an explicit dependency on python3?

Reply:
It should depend on Python 2, and 3.7 at build time. There are still two scripts that use Python 2 at runtime, but otherwise we just need 3.7 at build time to make the virtualenv.